pkgs/by-name/pi/piliplus/package.nix

```diff
@@ -0,0 +1,107 @@
+{
+  lib,
+  fetchFromGitHub,
+  flutter335,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  git,
+  mpv-unwrapped,
+  libplacebo,
+}:
+
+let
+  version = "1.1.4.1";
+  description = "Third-party Bilibili client developed in Flutter";
+in
+flutter335.buildFlutterApplication {
+  pname = "piliplus";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "bggRGjQaUbCoE";
+    repo = "PiliPlus";
+    tag = version;
+    hash = "sha256-sJKYfRoNmArHBsaHN9pARH9G6RrkRnRXDd28g2SezeU=";
+    leaveDotGit = true;
```

Comment:
Honestly, we shouldn't be doing this. Since another comment indicates that `lib/scripts/build.sh` needs this, I think we can patch it out.

Reply:
You could just use `rev` and use the commit hash instead of the tag. It's often better actually since many packages will replace tags which is a no no for reproducibility.